build-system/tasks/release/index.js

```diff
@@ -444,6 +454,9 @@ async function release() {
   await prependConfig_(outputDir);
 
   if (!argv.flavor || argv.flavor == 'base') {
+    log('Compiling 3p vendor configs');
+    await compile3pVendors_();
```

Comment:
This will increase build time during release by a large factor (4x I think, because we're going from 170 extensions to 170 extensions + 2*270 vendor configs). Looping in @danielrozenberg for comment on how to parallelize this during release.

Reply:
How about splitting this into its own separate flavor:
```js
if (!argv.flavor || argv.flavor == 'vendors') {
    log('Compiling 3p vendor configs');
    await compile3pVendors_();
}
```

It should be a small change to the release runner to add this extra flavor on each release